react/src/views/Transfers/TransferAgreementOverview/TransferAgreementOverviewView.tsx

```diff
@@ -0,0 +1,370 @@
+            agreementRow.partnerOrg = element.sourceOrganisation.name;
+          }
+        } else if (currentOrgId === sourceOrgId) {
+          // we are can only send stock to the partner
+          agreementRow.direction = TransferAgreementType.SendingTo;
+          agreementRow.partnerOrg = element.targetOrganisation.name;
+        } else if (currentOrgId === targetOrgId) {
+          // we are can only receive items from the partner
+          agreementRow.direction = TransferAgreementType.ReceivingFrom;
+          agreementRow.partnerOrg = element.sourceOrganisation.name;
+        }
+
+        // The initiating Org is not always the sourceOrg. Therefore, we need to do this complex if statement
+        if (
+          element.state === TransferAgreementState.UnderReview &&
+          ((currentOrgId === targetOrgId &&
+            (element.type === TransferAgreementType.Bidirectional ||
+              element.type === TransferAgreementType.SendingTo)) ||
+            (currentOrgId === sourceOrgId && element.type === TransferAgreementType.ReceivingFrom))
+        ) {
+          // You can accept this agreement if it is UnderReview and the partnerOrg created it
+          agreementRow.state = CanAcceptTransferAgreementState.CanAccept;
+        }
+
+        // prepare shipment data
+        const shipmentsTmp = [] as IShipmentBase[];
+        element.shipments.forEach((shipment) => {
+          if (globalPreferences.availableBases !== undefined) {
+            if (
+              shipment.targetBase != null &&
+              globalPreferences.availableBases.findIndex(
+                ({ id }) => shipment.targetBase?.id === id,
+              ) === -1
+            ) {
+              shipmentsTmp.push(shipment.targetBase);
+            } else if (
+              shipment.sourceBase != null &&
+              globalPreferences.availableBases.findIndex(
+                ({ id }) => shipment.sourceBase?.id === id,
+              ) === -1
+            ) {
+              shipmentsTmp.push(shipment.sourceBase);
+            }
+          }
+        });
+
+        agreementRow.shipments = shipmentsTmp.reduce((acc, { id, name }) => {
+          const count = acc[id] ? acc[id].count + 1 : 1;
+          acc[id] = { id, name, count };
+          return acc;
+        }, {});
+
+        return agreementRow;
+      }
+      return undefined;
+    }) || [];
+
+  // Define columns
+  const columns = useMemo(
+    () => [
+      {
+        Header: "",
+        accessor: "direction",
+        Cell: DirectionCell,
+        Filter: SelectColumnFilter,
+        filter: "includesSome",
+      },
+      {
+        Header: "Partner",
+        accessor: "partnerOrg",
+        disableFilters: true,
+      },
+      {
+        Header: "Status",
+        accessor: "state",
+        // eslint-disable-next-line react/no-unstable-nested-components
+        Cell: ({ ...cellProps }) => (
+          <StatusCell onClick={openTransferAgreementOverlay} {...cellProps} />
```

Comment:
I find it visually more pleasing if the button spanned the entire cell but I'll Roanna make a comment on that once it's in staging